### PR TITLE
Added the benefit of deploy to knative service

### PR DESCRIPTION
**Pull request by gyliu513**
@markusthoemmes @evankanderson ^^

### CODE REVIEW

**Review by snneji — CHANGES_REQUESTED** [review]
Hi, I've added a few suggestions. 

I also think it would make sense to move this "Benefits" section to the top of the page, just before "Determine if your workload is a good fit for Knative". This is so that users can easily see the info they need to help them decide whether converting to a Knative Service is something they want to do.

File: docs/serving/convert-deployment-to-knative-service.md [hunk 1]
```diff
@@ -69,3 +69,8 @@ spec:
         ports:
         - containerPort: 80
 ```
+
+## Benefits
+
```
[the remark this answers is not in the document]
Comment (gyliu513) [reply docs/serving/convert-deployment-to-knative-service.md:74]: Done

File: docs/serving/convert-deployment-to-knative-service.md [hunk 2]
```diff
@@ -69,3 +69,8 @@ spec:
...
+- Reduce footprint of the service instance as the instance will be scale to 0 when it becomes idle.
```
[the remark this answers is not in the document]
Comment (gyliu513) [reply docs/serving/convert-deployment-to-knative-service.md:75]: Done

File: docs/serving/convert-deployment-to-knative-service.md [hunk 3]
```diff
@@ -69,3 +69,8 @@ spec:
...
+- Performance can be improved via build-in autoscaling for knative service.
```
[the remark this answers is not in the document]
Comment (gyliu513) [reply docs/serving/convert-deployment-to-knative-service.md:76]: Done

**Review by gyliu513 — COMMENTED** [review]
Thanks @snneji

**Review by abrennan89 — APPROVED** [review]

**Pull request merged**